PocketKit/Sources/Textile/Tags/Components/InputTagsView.swift

```diff
@@ -4,17 +4,29 @@
 
 import SwiftUI
 
-struct InputTagsView: View {
+public struct InputTagsView: View {
     let tags: [String]
     let removeTag: (String) -> Void
     let upsellView: AnyView
+    let geometry: GeometryProxy
 
     @Namespace
     var animation
 
-    let geometry: GeometryProxy
+    enum Constants {
+        static let tagsHorizontalSpacing: CGFloat = 6
+        static let tagPadding: CGFloat = 4
+        static let frameSize = CGSize(width: 5, height: 5)
```

Comment:
Minor: How do these values handle Dynamic Type?

Reply:
discussed via slack that it plays nice in terms of padding. If design wants to increase the frameSize for closed button can revisit during the design QA